app/models/email_address.rb

```diff
@@ -37,6 +37,15 @@ def gov_or_mil?
     email.end_with?('.gov', '.mil')
   end
 
+  def fed_agency_issues_piv?
+    email_domain = get_email_domain
+    IdentityConfig.store.federal_agencies_with_piv.include?(email_domain)
```

Comment:
As mentioned in the corresponding write-up, I think we may want to do this lookup from a database table rather than a configuration value, since there could be thousands of domains.

Similar to what we do with populating and looking up within the `DisposableDomain` table for disposable domains.